longest_palindromic_substring/longest_palindromic_substring.py

```diff
@@ -0,0 +1,46 @@
+def expand_around_center(test, left, right):
+    """
+    :param test: Input string whose substrings are to be checked
+    :param left: Left end of result palindromic substring
+    :param right: Right end of result palindromic substring
+    :return: Length of palindromic substring
+    """
+    n = len(test)
```

Comment:
Take a nice look, its a condition in while loop, so it will check more than that and calculating `len(test)` each time would take more time comparatively.